### PR TITLE
feat: card

### PR DESCRIPTION
**Pull request by bhtibrewal**
Types of card added 
- basic card 
- card with badge
- card with image 
- card with text over image
- horizontal card
- card with text overlay

### CODE REVIEW

File: src/components/card/card.html [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+        </div>
+        <div class="component-container">
+            
+            <div class="card w-40 basic">
+                <div class="content">
+                    <div>
+                        <h2 class="hero">Conceptual Art</h2>
+                    </div>
+                    <div class="card-header">
+                        <h1>Conceptual Art: Abstract or Reality</h1>
```
Comment (Ankur9669) [inline src/components/card/card.html:78]: You can give a little margin here as it is looking quite clumsy.

File: src/components/card/card.html [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+                    <button class="btn outline-btn-secondary">Remind me later</button>
+                </div>
+            </div>
+        
+            <div class="card w-40 basic">
+                <span class="card-badge">Popular</span>
+                <div class="content">
+                    <div>
+                        <h2 class="hero">Conceptual Art</h2>
+                    </div>
```
Comment (Ankur9669) [inline src/components/card/card.html]: You can remove this div which is wrapping h2 element.

File: src/components/card/card.html [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+
+            <p>You can also replace the hero element with a img of your choice.<br>
+                To get card with text over image just replace the class <span class="highlight-text">basic</span> with the class <span class="highlight-text">text-o-media</span></p>
+        </div>
+        <div class="component-container">
+            <div class="card w-40 basic">
+                <div class="content">
+                    <img class="card-img"
+                        src="https://render.fineartamerica.com/images/images-new-artwork/images-medium-5/la-porta-rossa-sulla-salita-guido-borelli.jpg?v=2"
+                        alt="" />
```
Comment (Ankur9669) [inline src/components/card/card.html:132]: You have an empty alt text.

File: src/components/card/card.html [hunk 4]
```diff
@@ -0,0 +1,383 @@
...
+                </div>
+            </div>
+        </div>
+        </div>
+
+        <!-- horizontal card -->
+        <h3>Horizontal Card</h3>
+        <p>
+
+        </p>
```
Comment (Ankur9669) [inline src/components/card/card.html:218]: You can use br tag if you wanna have space instead of p

File: src/components/card/card.html [hunk 5]
```diff
@@ -0,0 +1,383 @@
...
+                            <path
+                                d="M7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2s-.9-2-2-2zM1 2v2h2l3.6 7.59l-1.35 2.45c-.16.28-.25.61-.25.96c0 1.1.9 2 2 2h12v-2H7.42c-.14 0-.25-.11-.25-.25l.03-.12l.9-1.63h7.45c.75 0 1.41-.41 1.75-1.03l3.58-6.49A1.003 1.003 0 0 0 20 4H5.21l-.94-2H1zm16 16c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2s2-.9 2-2s-.9-2-2-2z"
+                                fill="currentColor"></path>
+                        </svg>
+                        add to cart
+                    </button>
+                    <button class="btn outline-btn-secondary">move to wishlist</button>
+
+
+
```
Comment (Ankur9669) [inline src/components/card/card.html:261]: You can remove white spaces.

File: src/components/card/card.css [hunk 6]
```diff
@@ -0,0 +1,176 @@
+.card {
+  background: var(--light-bg);
+  border-radius: 1rem;
+  border: thin solid var(--gray-bg);
+  margin: auto;
+  
+  overflow: hidden;
```
Comment (Ankur9669) [inline src/components/card/card.css:7]: You can remove white space from here

File: src/components/card/card.css [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
+
+/* Horizontal Image and Text */
+
+.hori .content {
+  display: flex;
+  flex-direction: row;
+}
+
+.hori img {
+  width: 15rem;
```
Comment (Ankur9669) [inline src/components/card/card.css:186]: You can use object-fit property here.

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
Great code, doing these changes would make your code even better.

**Comment by Naman-Saxena1** [comment]
Nice work, you can add media query to your left sidebar and reduce its width for smaller screens. Also set some min-width for your content, as it is shrinking to letters in responsive mode. And you have not done 1) Card with dismiss and 2) Card with shadow according to LMS PRD.

**Pull request merged**